.github/workflows/postsubmit.yml

```
@@ -8,65 +8,84 @@
-      - name: Run fmt
+      - name: Check formatting
         run: cargo fmt --check
 
-      - name: Run clippy
+      - name: Run clippy linter
         run: cargo clippy --no-deps
 
-      - name: Machete
+      - name: Check for unused dependencies
         uses: bnjbvr/cargo-machete@main
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)